packages/styles/card.css

```diff
@@ -75,10 +75,6 @@
   box-sizing: border-box;
 }
 
-.Card__content ul li {
```

Comment:
Do we have any list inside .Card__content that actually requires the border? I don't recall any, but maybe we should check with @awpearlm

Reply:
I asked @schne324 about this and he doesn't know, but regardless this is way too generic of a selector. If we intended to have some sort of list components inside of a card we need something more specific.